apis/v1alpha1/httproute_types.go

```diff
@@ -64,11 +64,12 @@ type HTTPRouteSpec struct {
 	//
 	// Hostname can be "precise" which is a domain name without the terminating
 	// dot of a network host (e.g. "foo.example.com") or "wildcard", which is
-	// a domain name prefixed with a single wildcard label (e.g. "*.example.com").
-	// The wildcard character '*' must appear by itself as the first DNS
+	// a domain name prefixed with a single wildcard label (e.g. "&ast;.example.com").
+	// The wildcard character '&ast;' must appear by itself as the first DNS
 	// label and matches only a single label.
-	// You cannot have a wildcard label by itself (e.g. Host == "*").
+	// You cannot have a wildcard label by itself (e.g. Host == "&ast;").
```

Comment:
Just trying to capture all changes from "*" in case we miss any.

Reply:
dropped